main.go

```diff
@@ -0,0 +1,67 @@
+package main
+
+// Users will give us bulk correction files following https://tools.ietf.org/html/draft-google-self-published-geofeeds-02
+// It isn't uncommon for the corrections they list to either match what we already have
+// or to be worse. This script can help us work that out. Right now, it only looks
+// at the ISO country code, but checking more fields should be easy enough.
+import (
+	"fmt"
+	"github.com/oschwald/geoip2-golang"
+	"github.com/yunabe/easycsv"
+	"log"
+	"net"
+	"os"
+	"strings"
+)
+
+// Usage Example:
+// go run go/check_csv_corrections/main.go /path/to/corrections.csv /path/to/mmdbfile.mmdb
+func main() {
+	csvFilename := os.Args[1]
```

Comment:
Strongly suggest checking the length of `os.Args` before attempting to access its elements, and printing out an error message and exiting if it's not long enough.

```
nchelluri@grugbarn:~/dev/mm-geofeed-verifier [klp2/initial-commits=] $ ./mm-geofeed-verifier 
panic: runtime error: index out of range

goroutine 1 [running]:
main.main()
	/home/nchelluri/dev/mm-geofeed-verifier/main.go:20 +0x409
```